src/extension.ts

```diff
@@ -215,6 +224,15 @@ export function activate(context: vscode.ExtensionContext): void {
                     if (newApp.id === node.id) {
                         // if the node selected for deployment is the same newly created nodes, stifle the confirmDeployment dialog
                         confirmDeployment = false;
+                        newApp.treeItem.client.getSiteConfig().then(
+                            (createdAppConfig: SiteConfigResource) => {
+                                this.properties.os = createdAppConfig.linuxFxVersion ? 'Linux' : 'Windows';
```

Comment:
nit: if "os" is just based on linuxFxVersion and we're already collecting linuxFxVersion, do we need a separate 'os' field?

Reply:
Agreed that we don't need it, but might make it more intuitive to read the telemetry especially for people without context.  But if you think it's redundant, I'm cool with removing it.

Reply:
Yeah I guess I prefer to mess with the query in the telemetry dashboard to make it more readable, rather than trying to do extra stuff directly in the code here